test/integration/serving/serving_test.go

```diff
@@ -32,7 +32,7 @@ import (
 	"k8s.io/apiserver/pkg/server/options"
 	"k8s.io/cloud-provider"
 	"k8s.io/cloud-provider/fake"
-	cloudctrlmgrtesting "k8s.io/kubernetes/cmd/cloud-controller-manager/app/testing"
+	cloudctrlmgrtesting "k8s.io/cloud-provider/testing"
```

Comment:
Would it make sense to keep this in under `app` based on it's previous location (i.e. `k8s.io/cloud-provider/app/testing`)?